docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,62 @@
+<!--[metadata]>
```

Comment:
FWIW this section might be a decent place to bring up `--opt secure` option of creating overlay networks.  Communication between them is not encrypted by default IIRC and so implying that they are totally secure (i.e., not MITM-able) is pretty strong.